userguide/sharing.rst

```diff
@@ -314,10 +316,11 @@ the :guilabel:`Shared` sidebar.
 Creating Authenticated and Time Machine Shares
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-macOS includes the Time Machine application which can be used to
-schedule automatic backups.  In this configuration example, a Time
+macOS includes the Time Machine feature which can be used to
+perform automatic backups.  In this configuration example, a Time
 Machine user will be configured to backup to an AFP share on a
-%brand% system. Creating a separate Time Machine
+%brand% system. (%brand% does not yet support Time Machine over SMB.)
+Creating a separate Time Machine
```

Comment:
"A separate Time Machine share is recommended for each user backing up their system."